content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,471 @@
+
+      - Within EU: the *buyer* integrates the invoice received with the **VAT information**
+        due in Italy (i.e., **vendor bill tax integration**);
+      - Non-EU: the *buyer* sends themselves an invoice (i.e., **self-billing**).
+
+      Odoo exports a transaction as `TD17` if the following conditions are met:
+
+      - Is a vendor bill;
+      - At least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+      - All invoice lines either have :guilabel:`Services` as **products**, or a tax with the
+        :guilabel:`Services` as **tax scope**.
+
+  .. tab:: `TD18`
+
+     Buying **goods** from **EU**:
+
+     Invoices issued within the EU follow a **standard format**, therefore only an integration of
+     the existing invoice is required.
+
+     Odoo exports a transaction as `TD18` if the following conditions are met:
+
+     - Is a vendor bill;
+     - At least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+     - All invoice lines either have :guilabel:`Consumable` as **products**, or a tax with the
+       :guilabel:`Goods` as **tax scope**.
+
+  .. tab:: `TD19`
+
+     Buying **goods** from a **foreign** vendor, but the **goods** are already in **Italy** in a
+     **VAT deposit**:
+
+     - From EU: the *buyer* integrates the invoice received with the **VAT information** due in
+       Italy (i.e., **vendor bill tax integration**);
+     - Non-EU: the *buyer* sends an invoice to *themselves* (i.e., **self-billing**).
+
+     Odoo exports a move as a `TD19` if the following conditions are met:
+
+     - Is a vendor bill;
+     - At least one tax on the invoice lines targets the tax grid :ref:`VJ3 <italy/grids>`;
+     - All invoice lines either have :guilabel:`Consumable` products, or a tax with
+       :guilabel:`Goods` as tax scope.
+
+.. _italy/internal-reverse:
+
+Internal reverse charge
+-----------------------
+
+.. warning::
+   Odoo currently does not support **internal reverse charge** processes.
+
+In some cases, vendor bills have the VAT duty **reverse charged** on the *buyer* for **national
+purchases** too (e.g., cleaning company invoices, construction sub-contractor invoices, scrap
+selling, tablets and mobile phones, etc.). For these operations, the *seller* or *provider* issues
+an electronic invoice with :guilabel:`Tipo Documento` code `TD01`, `TD02` for down payments, or
+`TD04` for credit notes, with the **tax-exemption code** ranging from `N6.1` to `N6.9`.
+
+The *buyer* is required to issue themselves an invoice (i.e., **self-billing**) for the VAT due with
+:guilabel:`Tipo Documento` code `TD16`, which is sent through the :abbr:`SdI (Systema di
+Interscambio)` with the buyer *themselves* as **recipient** (using their own :guilabel:`Codice
+Destinatario`). That way, they can deduct their VAT. The **self-billed** invoice must be issued and
+recorded in the VAT sales register within **15 days** from receipt.
```

Comment:
```suggestion
Reverse charge within Italy
```
That should be clear enough.

Reply:
Not sure if you meant to delete the all paragraph, as Tom wrote it and it's pretty clear to me, I'll leave it.